frontend/appflowy_tauri/src/appflowy_app/components/user/application/notifications/user_listener.ts

```diff
@@ -10,7 +10,7 @@ export class UserNotificationListener extends AFNotificationListener<UserNotific
   onUserSignIn?: OnUserSignIn;
 
   constructor(params: {
-    userId?: String;
+    userId?: string;
```

Comment:
What different between String and string?

Reply:
https://typescript-eslint.io/rules/ban-types/

The 'string' is more correct.

Reply:
Got it